openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -66,7 +66,7 @@ export class Carousel {
             speed: 300,
             slidesToShow: this.config.booksPerBreakpoint[0],
             slidesToScroll: this.config.booksPerBreakpoint[0],
-            responsive: [1200, 1024, 600, 480, 360]
+            responsive: [1200, 1024, 740, 580, 360]
```

Comment:
```suggestion
            responsive: [1200, 1024, 600, 480, 360]
```
Lets restore these (I think that's what Drini wanted) then I'll put it on testing so we can see how things look!